core/commands/UserResetPasswordCommand.php

```diff
@@ -33,6 +33,13 @@
  * }
  */
 class UserResetPasswordCommand extends Command {
+	/**
+	 * Constants for execute() method's return value.
+	 */
+	const RESULT_FAILURE = 0;
+	const RESULT_RESET = 1;
```

Comment:
Use meaningful strings that we can return to the REST API client in the response and not magic numbers over the wire.